File: A_Too_Min_Too_Max.py
import sys
input=sys.stdin.buffer.readline
for _ in range(int(input())):
    n=int(input())
    arr=list(map(int,input().split()))
    arr.sort(reverse=True)
    ans=0
    a=arr[0]
    b=arr[-1]
    c=arr[1]
    d=arr[-2]
    print(abs(a-b)+abs(b-c)+abs(c-d)+abs(d-a))

File: A_Too_Min_Too_Max.cpp
#include<bits/stdc++.h>
using namespace std;
#define w(t) int t; cin>>t; while(t--)
#define nn "\n"
#define int long long
#define pb push_back
#define ld long double
int32_t main(){
    w(t){
        int n; cin>>n;
        int a[n];
        for(auto &it: a) cin>>it;
        sort(a,a+n);
        vector<int>v={a[0],a[n-1],a[1],a[n-2]};
        int ans=0;
        for(int i=0; i<4; i++){
            ans+=abs(v[i]-v[(i+1)%4]);
        }
        cout<<ans<<nn;
    }
}